Fix eslint with prettier

Recent PR added linting errors for indentation, ie:
```
$ npm run lint

> spring-cloud-dataflow-ui@3.0.0 lint /home/jvalkealahti/repos/spring/spring-cloud-dataflow-ui/ui
> ng lint


Linting "dataflow"...

/home/jvalkealahti/repos/spring/spring-cloud-dataflow-ui/ui/src/app/shared/api/job.service.ts
  66:1  error  Expected indentation of 6 spaces but found 8  @typescript-eslint/indent
  67:1  error  Expected indentation of 4 spaces but found 6  @typescript-eslint/indent

```
https://stackoverflow.com/questions/56337176/prettier-and-eslint-indents-not-working-together

--- a/ui/.eslintrc.json
+++ b/ui/.eslintrc.json
@@ -14,7 +14,8 @@
       ],
       "parserOptions": {
         "project": [
-          "tsconfig.json"
+          "src/tsconfig.app.json",
+          "src/tsconfig.spec.json"
         ],
         "createDefaultProgram": true
       },
@@ -57,10 +58,7 @@
             "accessibility": "explicit"
           }
         ],
-        "@typescript-eslint/indent": [
-          "error",
-          2
-        ],
+        "@typescript-eslint/indent": "off",
         "@typescript-eslint/member-delimiter-style": [
           "error",
           {

--- a/ui/angular.json
+++ b/ui/angular.json
@@ -167,9 +167,9 @@
         "lint": {
           "builder": "@angular-eslint/builder:lint",
           "options": {
-            "tsConfig": "e2e/tsconfig.e2e.json",
-            "exclude": [
-              "**/node_modules/**"
+            "lintFilePatterns": [
+              "e2e/src/**/*.ts",
+              "e2e/src/**/*.html"
             ]
           }
         }

--- a/ui/e2e/src/app.e2e-spec.ts
+++ b/ui/e2e/src/app.e2e-spec.ts
@@ -1,15 +1,14 @@
-import { browser, element } from 'protractor';
-import { AppsPage } from './page/apps/apps.po';
-import { ManageRecordsPage } from './page/manage/records.po';
-import { ManageImportExportPage } from './page/manage/import-export.po';
-import { StreamsListPage } from './page/streams/list.po';
-import { StreamsRuntimePage } from './page/streams/runtime.po';
-import { TasksJobsTasksPage } from './page/tasks-jobs/tasks.po';
-import { TasksJobsExecutionsPage } from './page/tasks-jobs/executions.po';
-import { TasksJobsJobExecutionsPage } from './page/tasks-jobs/job-executions.po';
+import {browser, element} from 'protractor';
+import {AppsPage} from './page/apps/apps.po';
+import {ManageRecordsPage} from './page/manage/records.po';
+import {ManageImportExportPage} from './page/manage/import-export.po';
+import {StreamsListPage} from './page/streams/list.po';
+import {StreamsRuntimePage} from './page/streams/runtime.po';
+import {TasksJobsTasksPage} from './page/tasks-jobs/tasks.po';
+import {TasksJobsExecutionsPage} from './page/tasks-jobs/executions.po';
+import {TasksJobsJobExecutionsPage} from './page/tasks-jobs/job-executions.po';
 
 describe('should display the pages', () => {
-
   let appsPage: AppsPage;
   let manageRecordsPage: ManageRecordsPage;
   let manageImportExportPage: ManageImportExportPage;
@@ -34,76 +33,66 @@ describe('should display the pages', () => {
   });
 
   describe('Apps', () => {
-
-    it('should display the apps page', async (done) => {
+    it('should display the apps page', async done => {
       await appsPage.navigateTo();
       const title = await appsPage.getTitle();
       expect(element(title).getText()).toEqual('Applications');
       done();
     });
-
   });
 
   describe('Manage section', () => {
-
-
-    it('should display the record page', async (done) => {
+    it('should display the record page', async done => {
       await manageRecordsPage.navigateTo();
       const title = await manageRecordsPage.getTitle();
       expect(element(title).getText()).toEqual('Audit Records');
       done();
     });
 
-    it('should display the Import / Export page', async (done) => {
+    it('should display the Import / Export page', async done => {
       await manageImportExportPage.navigateTo();
       const title = await manageImportExportPage.getTitle();
       expect(element(title).getText()).toEqual('Import / Export');
       done();
     });
-
   });
 
   describe('Streams section', () => {
-
-    it('should display the stream page', async (done) => {
+    it('should display the stream page', async done => {
       await streamsListPage.navigateTo();
       const title = await streamsListPage.getTitle();
       expect(element(title).getText()).toEqual('Streams');
       done();
     });
 
-    it('should display the runtime page', async (done) => {
+    it('should display the runtime page', async done => {
       await streamsRuntimePage.navigateTo();
       const title = await streamsRuntimePage.getTitle();
       expect(element(title).getText()).toEqual('Runtime');
       done();
     });
-
   });
 
   describe('Tasks section', () => {
-
-    it('should display the task page', async (done) => {
+    it('should display the task page', async done => {
       await tasksJobsTasksPage.navigateTo();
       const title = await tasksJobsTasksPage.getTitle();
       expect(element(title).getText()).toEqual('Tasks');
       done();
     });
 
-    it('should display the task executions page', async (done) => {
+    it('should display the task executions page', async done => {
       await tasksJobsExecutionsPage.navigateTo();
       const title = await tasksJobsExecutionsPage.getTitle();
       expect(element(title).getText()).toEqual('Task Executions');
       done();
     });
 
-    it('should display the job executions page', async (done) => {
+    it('should display the job executions page', async done => {
       await tasksJobsJobExecutionsPage.navigateTo();
       const title = await tasksJobsJobExecutionsPage.getTitle();
       expect(element(title).getText()).toEqual('Job Executions');
       done();
     });
-
   });
-
 });

--- a/ui/e2e/src/page/apps/apps.po.ts
+++ b/ui/e2e/src/page/apps/apps.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class AppsPage {
-
   navigateTo() {
     return browser.get(`#/apps`);
   }
@@ -10,5 +9,4 @@ export class AppsPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/manage/import-export.po.ts
+++ b/ui/e2e/src/page/manage/import-export.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class ManageImportExportPage {
-
   navigateTo() {
     return browser.get(`#/manage/import-export`);
   }
@@ -10,5 +9,4 @@ export class ManageImportExportPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/manage/records.po.ts
+++ b/ui/e2e/src/page/manage/records.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class ManageRecordsPage {
-
   navigateTo() {
     return browser.get(`#/manage/records`);
   }
@@ -10,5 +9,4 @@ export class ManageRecordsPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/streams/list.po.ts
+++ b/ui/e2e/src/page/streams/list.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class StreamsListPage {
-
   navigateTo() {
     return browser.get(`#/streams/list`);
   }
@@ -10,5 +9,4 @@ export class StreamsListPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/streams/runtime.po.ts
+++ b/ui/e2e/src/page/streams/runtime.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class StreamsRuntimePage {
-
   navigateTo() {
     return browser.get(`#/streams/runtime`);
   }
@@ -10,5 +9,4 @@ export class StreamsRuntimePage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/tasks-jobs/executions.po.ts
+++ b/ui/e2e/src/page/tasks-jobs/executions.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class TasksJobsExecutionsPage {
-
   navigateTo() {
     return browser.get(`#/tasks-jobs/task-executions`);
   }
@@ -10,5 +9,4 @@ export class TasksJobsExecutionsPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/tasks-jobs/job-executions.po.ts
+++ b/ui/e2e/src/page/tasks-jobs/job-executions.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class TasksJobsJobExecutionsPage {
-
   navigateTo() {
     return browser.get(`#/tasks-jobs/job-executions`);
   }
@@ -10,5 +9,4 @@ export class TasksJobsJobExecutionsPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/page/tasks-jobs/tasks.po.ts
+++ b/ui/e2e/src/page/tasks-jobs/tasks.po.ts
@@ -1,8 +1,7 @@
-import { browser, by } from 'protractor';
-import { ElementHelper } from '../../utils/element.helper';
+import {browser, by} from 'protractor';
+import {ElementHelper} from '../../utils/element.helper';
 
 export class TasksJobsTasksPage {
-
   navigateTo() {
     return browser.get(`#/tasks-jobs/tasks`);
   }
@@ -10,5 +9,4 @@ export class TasksJobsTasksPage {
   getTitle() {
     return ElementHelper.getElement(by.css('.content-area h1'));
   }
-
 }

--- a/ui/e2e/src/utils/element.helper.ts
+++ b/ui/e2e/src/utils/element.helper.ts
@@ -1,45 +1,41 @@
-import { browser, element, ElementFinder, ProtractorBy, ElementArrayFinder } from 'protractor';
-import { protractor } from 'protractor/built/ptor';
-import { By } from 'selenium-webdriver';
+import {browser, element, ElementFinder, ProtractorBy, ElementArrayFinder} from 'protractor';
+import {protractor} from 'protractor/built/ptor';
+import {By} from 'selenium-webdriver';
 
 const EC = protractor.ExpectedConditions;
 
 const defaultTimeout = 30000;
 
 export class ElementHelper {
-
   public static async u(e: By): Promise<number> {
     return element.all(await ElementHelper.getElement(e)).count();
   }
 
   public static async getElement(byValue: By): Promise<By> {
     const conditionA = EC.visibilityOf(element(byValue));
-    return browser.wait(
-      EC.and(conditionA), defaultTimeout)
-      .then(() => byValue, () => {
+    return browser.wait(EC.and(conditionA), defaultTimeout).then(
+      () => byValue,
+      () => {
         fail(`No visible emelement for '${byValue}'.`);
         return byValue;
-      });
+      }
+    );
   }
 
   public static getClickableElementOrWait(by: By) {
     const conditionA = EC.visibilityOf(element(by));
-    return browser.wait(
-      EC.and(conditionA), defaultTimeout, 'getClickableElementOrWait failed.')
+    return browser
+      .wait(EC.and(conditionA), defaultTimeout, 'getClickableElementOrWait failed.')
       .then(() => element(by));
   }
 
   public static async waitForSpinners() {
     const conditionB = EC.invisibilityOf(element(By.css('.app-blocker')));
     const conditionC = EC.invisibilityOf(element(By.css('modal-body')));
-    await browser.wait(() => {
-      return EC.and(conditionB, conditionC);
-    }, defaultTimeout, 'waitForSpinners failed.');
+    await browser.wait(() => EC.and(conditionB, conditionC), defaultTimeout, 'waitForSpinners failed.');
   }
 
-
   public static async clickElement(by: By, clickFirst: boolean, parent: ElementFinder): Promise<boolean> {
-
     console.log('Clicking ' + by);
 
     let returnValue = false;
@@ -67,7 +63,7 @@ export class ElementHelper {
 
         if (clickFirst) {
           const t = parent.all(by);
-          if (await t.count() > 0) {
+          if ((await t.count()) > 0) {
             clickableElement = t.first();
           } else {
             returnValue = false;
@@ -78,15 +74,17 @@ export class ElementHelper {
 
         console.log(`Attempt ${attempt} for clickableElement:` + clickableElement);
 
-        await clickableElement.click().then(() => {
-          console.log(`Successfully Clicked: ${by} (Attempt: ${attempt})`);
-          clearTimeout(timeout);
-          returnValue = true;
-        }, () => {
-          console.log(`Failed click: ${by} (Attempt: ${attempt})`);
-          returnValue = false;
-        });
-
+        await clickableElement.click().then(
+          () => {
+            console.log(`Successfully Clicked: ${by} (Attempt: ${attempt})`);
+            clearTimeout(timeout);
+            returnValue = true;
+          },
+          () => {
+            console.log(`Failed click: ${by} (Attempt: ${attempt})`);
+            returnValue = false;
+          }
+        );
       }
     });
 
@@ -99,5 +97,4 @@ export class ElementHelper {
     await browser.wait(EC.elementToBeClickable(clickableElement), 15000);
     await clickableElement.click();
   }
-
 }

